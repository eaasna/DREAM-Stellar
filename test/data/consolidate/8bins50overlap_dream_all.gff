3	Stellar	eps-matches	1	176	94.382	+	.	1;seq2Range=686,863;cigar=169M1I4M1I3M;mutations=49G,57C,71C,89C,97T,165A,170A,173G,175T,176T
2	Stellar	eps-matches	187	386	94.0594	+	.	1;seq2Range=662,863;cigar=193M1I4M1I3M;mutations=2T,21C,73G,81C,95C,113C,121T,189A,194A,197G,199T,200T
2	Stellar	eps-matches	187	386	94.0594	+	.	1;seq2Range=662,863;cigar=193M1I4M1I3M;mutations=2T,21C,73G,81C,95C,113C,121T,189A,194A,197G,199T,200T
2	Stellar	eps-matches	68	258	94.3005	+	.	1;seq2Range=446,635;cigar=3M1I171M1I1M1D9M1D2M1D2M;mutations=4A,46C,63G,96C,105T,135G,168C,176C
2	Stellar	eps-matches	148	202	94.7368	+	.	1;seq2Range=912,968;cigar=53M1I1M1I1M;mutations=26A,54T,56T
2	Stellar	eps-matches	187	386	94.0594	+	.	1;seq2Range=662,863;cigar=193M1I4M1I3M;mutations=2T,21C,73G,81C,95C,113C,121T,189A,194A,197G,199T,200T
2	Stellar	eps-matches	68	258	94.3005	+	.	1;seq2Range=446,635;cigar=2M1I172M1I1M1D9M1D2M1D2M;mutations=3A,46C,63G,96C,105T,135G,168C,176C
2	Stellar	eps-matches	18	147	94.6153	+	.	1;seq2Range=1054,1183;cigar=130M;mutations=2C,14A,24A,35T,41T,54A,65G
2	Stellar	eps-matches	148	202	94.7368	+	.	1;seq2Range=912,968;cigar=53M1I1M1I1M;mutations=26A,54T,56T
1	Stellar	eps-matches	469	560	94.5652	+	.	1;seq2Range=60,151;cigar=92M;mutations=15G,58C,60A,87A,90G
1	Stellar	eps-matches	177	359	94.0217	+	.	1;seq2Range=1807,1987;cigar=92M1D1M1I83M2D4M;mutations=54C,70A,84T,94G,109A,139C,150G,163C
1	Stellar	eps-matches	272	426	94.1935	+	.	1;seq2Range=2107,2261;cigar=155M;mutations=3A,4G,8A,13A,23C,44C,52C,101C,138G
1	Stellar	eps-matches	177	359	94.0217	+	.	1;seq2Range=1807,1987;cigar=92M1D1M1I83M2D4M;mutations=54C,70A,84T,94G,109A,139C,150G,163C
1	Stellar	eps-matches	126	307	94.0217	+	.	1;seq2Range=204,384;cigar=2M1D2M1D171M2I1M1D3M;mutations=19A,24T,96G,159C,166A,170A,176A,177T
1	Stellar	eps-matches	26	205	94.4444	+	.	1;seq2Range=1199,1377;cigar=177M1D2M;mutations=2G,42T,56G,61T,107C,120A,137C,169T,172A
1	Stellar	eps-matches	272	426	94.1935	+	.	1;seq2Range=2107,2261;cigar=155M;mutations=3A,4G,8A,13A,23C,44C,52C,101C,138G
1	Stellar	eps-matches	177	359	94.0217	+	.	1;seq2Range=1807,1987;cigar=92M1D1M1I83M2D4M;mutations=54C,70A,84T,94G,109A,139C,150G,163C
1	Stellar	eps-matches	126	307	94.0217	+	.	1;seq2Range=204,384;cigar=2M1D2M1D171M2I1M1D3M;mutations=19A,24T,96G,159C,166A,170A,176A,177T
1	Stellar	eps-matches	26	205	94.4444	+	.	1;seq2Range=1199,1377;cigar=177M1D2M;mutations=2G,42T,56G,61T,107C,120A,137C,169T,172A
1	Stellar	eps-matches	72	125	94.4444	+	.	1;seq2Range=1640,1693;cigar=54M;mutations=3T,11G,27T
