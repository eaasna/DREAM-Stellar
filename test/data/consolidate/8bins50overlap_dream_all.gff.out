I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_7_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_7_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 3.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  3
  3, complement

# Eps-matches     : 1


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_6_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_6_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 2.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  2
  2, complement

# Eps-matches     : 1


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_5_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_5_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 2.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  2
  2, complement

# Eps-matches     : 3


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_4_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_4_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 2.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  2
  2, complement

# Eps-matches     : 4


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_3_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_3_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 1.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
  1, complement

# Eps-matches     : 1


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_2_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_2_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 1.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
  1, complement

# Eps-matches     : 2


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_1_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_1_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 1.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
  1, complement

# Eps-matches     : 4


I/O options:
  database file   : multi_seq_ref.fasta
  query file      : /tmp/valik/stellar_call_iIVcaz/query_0_0.fasta
  alphabet        : dna5
  output file     : /tmp/valik/stellar_call_iIVcaz/query_0_0.fasta.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.06 = (3/50)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 12
  s^min        : 12
  threshold    : 3
  distance cut : 50
  delta        : 16
  overlap      : 3

Loaded 1 query sequence.
Loaded sequence 1.

All matches resulting from your search have an E-value of: 
        6.4259e-16 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
  1, complement

# Eps-matches     : 4


